### DISCUSSION TITLE
Any recommended settings related to security I should consider for running public Bitcoin Explorer?

### DISCUSSION THREAD

**Discussion by jaska087** [janoside/btc-rpc-explorer#544] [Q&A]
Hi.
Just set up Bitcoin Explorer instance and though why not make it public as an backup solution for those in need.

Any settings/enviroment variables I should consider changing before going public?

Cool software and really the original idea was set it for myself and lndash instance.

**Comment by janoside** [comment]
Glad you're enjoying the project and I think it's great to make it public for others' use (as I do on bitcoinexplorer.org). For reference, the settings I use for that public instance are [here](https://github.com/janoside/btc-rpc-explorer#demo-site-settings). However, I don't use the Bitcoin Core instance backing that explorer for any real funds. If you plan to use Bitcoin Core for real funds, you might want to keep the instance private. I primarily say this because I haven't considered that use case in much detail and can't vouch for the safety/security in such a configuration.
